strawberry/arguments.py

```diff
@@ -94,48 +89,46 @@ def __init__(
             _deprecated_UNSET if default is inspect.Parameter.empty else default
         )
 
-        if self._annotation_is_annotated(type_annotation):
-            self._parse_annotated()
+        try:
+            evaled_type = type_annotation.evaluate()
+        except NameError:
+            pass
```

Comment:
can we add a comment on when this might happen? (I'm assuming when using if TYPE_CHECKING)

Reply:
Also, try/except with pass can be rewritten as:

```python
with contextlib.suppress(NameError):
    evaled_type = type_annotation.evaluate()
```

Reply:
Good idea! I've added one now